flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/library/HITSAlgorithm.java

```diff
@@ -160,7 +240,7 @@ public void sendMessages(Vertex<K, Tuple2<DoubleValue, DoubleValue>> vertex) {
 			double iterationValueSum = 1.0;
 
 			if (getSuperstepNumber() > 1) {
-				iterationValueSum = Math.sqrt(((DoubleValue) getPreviousIterationAggregate("sumVertexValue")).getValue());
+				iterationValueSum = Math.sqrt(((DoubleValue) getPreviousIterationAggregate("updatedValueSum")).getValue());
 			}
 			for (Edge<K, EV> edge : getEdges()) {
 				K messageSource = getSuperstepNumber() % 2 == 1 ? edge.getSource() : edge.getTarget();
```

Comment:
Could you please explain what is this and the following line doing? The condition is the same in both.